tests/fixtures/qasm2/bell.qasm

```diff
@@ -1,5 +1,6 @@
 OPENQASM 2.0;
 include "qelib1.inc";
 qreg q[2];
+creg c[2];
```

Comment:
why is including `creg` necessary if `creg` is never referenced again?

Reply:
Added so that we can cover the `num_clbits` method. Will add a measurement.